app/retail/templates/email_unsubscribed.html

```diff
@@ -0,0 +1,52 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include 'shared/head.html' %}
+    <link rel="stylesheet" type="text/css" href="{% static "v2/css/unsubscribe_email.css" %}">
+  </head>
+  <body class="interior {{ active }} g-font-muli">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid background">
+      <div class="justify-content-left pt-4 pl-4">
+        <img class="img-fluid logo" src="{% static 'v2/images/emails/email-subscribe-logo.png' %}" alt="Unsubscribed mail successful">
+      </div>
+      <div class="justify-content-center">
```

Comment:
not sure but I think this class name had changed on the master branch

Reply:
I'll take a look and fix this

Reply:
classes still exists